tests/components/lifx/__init__.py

```diff
@@ -22,10 +22,11 @@
 class MockMessage:
     """Mock a lifx message."""
 
-    def __init__(self):
+    def __init__(self, **kwargs):
         """Init message."""
         self.target_addr = SERIAL
         self.count = 9
+        [setattr(self, k, v) for k, v in kwargs.items() if k != "callb"]
```

Comment:
Usually we don't use a list comprehension to mutate data

Reply:
I needed a way for `MockLifxCommand` to return specific values without having to define a class for each mocked command. If you have another way of achieving this, I'm all ears/eyes/fingers.